requests/models.py

```diff
@@ -672,6 +672,7 @@ def __iter__(self):
 
     @property
     def ok(self):
+        """Returns true if :attr:`status_code` is 'OK'."""
```

Comment:
Ouh. My bad then. I can change the docstring to:

`Returns true if :attr:status_code is 'OK' or 'redirect'.`

Would something like that be ok?